source/js/buyers-guide/homepage-c-slider.js

```diff
@@ -22,7 +22,13 @@ const MINIMUM_HAPPINESS_RATING = 25;
 const MAXIMUM_CREEPINESS_RATING = 80;
 
 // Sticky search bar
```

Comment:
Move the following to within the window scroll handler since there are already many similar var assignments happening in there.

```
// Sticky search bar
const SEARCH_BAR = document.getElementById("sticky-bar");

// Creepy-face container
const CREEPY_FACE = document.querySelector(".creep-o-meter-information");

// Category bar. Used for relocating the creepo-face on desktop.
const CATEGORY_BAR = document.getElementById("multipage-nav")
```

Reply:
We don't want them _inside_ the `scroll` handler as that requires every pixel-scrolled to re-lookup the element through the DOM API. I can move them just outside though so they're close, but not spamming the DOM with lookups every time the window scrolls a little bit.